tv_apps/smart-system/js/interactive_notifications.js

```diff
@@ -273,21 +260,23 @@
   proto.hideNotification = function in_hideNotification(type, msg, button) {
     if (this._activeType === type && this._activeMessage === msg) {
       this._banner.setAttribute('aria-hidden', 'true');
-      this._banner.classList.add('fading-out');
-      this._banner.close();
-      if (msg.onClosed) {
-        msg.onClosed(button);
-      }
-      this._activeType = null;
-      this._activeMessage = null;
-
-      if (!this.hasPendings() && window.AppWindowManager &&
-          AppWindowManager.getActiveApp()) {
-        // We need to stop KeyNavigator while we don't need it.
-        this._keyNavigator.stop();
-        // If there is active app, we need to focus it back.
-        AppWindowManager.getActiveApp().getTopMostWindow().focus();
-      }
+      this._banner.hide();
+      this._hideCallback = function () {
```

Comment:
There is no need to create _hideCallback function dynamically. We should make it as a member function of InteractiveNotification.

The only thing we need to take care is `button` variable. It should still work while we move the following code out of _hideCallback:
```
if (msg.onClosed) {
  msg.onClosed(button);
}
```